src/session_impl.cpp

```diff
@@ -588,30 +587,7 @@ namespace aux {
 		TORRENT_ASSERT(is_single_thread());
 
 #ifndef TORRENT_DISABLE_LOGGING
-		// this alert is a bit special. The stats headers aren't very useful
```

Comment:
I want to get rid of the posting, because of the noise. In my particular case, I have code that it's prepared to print "log alerts" always, and I control that by putting the right value in the mask (or not), but now I always get this "headers". No other problem.

I kind of like the benefits of a new alert, even at the cost of creating a "new alert" type. Let me work on it and let's see how it looks.

Thanks for taking your time to explain me the benefits of having the message in the log stream, now I understand better how this "messages logging" can be used.